frontend/cypress/support/commands.js

```diff
@@ -23,3 +23,9 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add("globalLogin", login => {
+  cy.get('#email').type(login.email);
```

Comment:
I think we should follow doc and create specific tags for cypress.
https://docs.cypress.io/guides/references/best-practices#How-It-Works

Reply:
```
- data-cy
- data-test
- data-testid
```